lib/bundler/cli.rb

```diff
@@ -465,10 +465,14 @@ def platform
     end
 
     desc "inject GEM VERSION", "Add the named gem, with version requirements, to the resolved Gemfile"
+    method_option "source", :type => :string, :banner =>
```

Comment:
Thanks for review @colby-swandale . I have edited the PR according to to your comments.